src/workers/algorithm/index.ts

```diff
@@ -20,7 +20,6 @@ export async function run(...args) {
 
     worker.addEventListener('error', (error) => {
       reject(error)
-      console.error('Worker error:', error)
```

Comment:
This console.error is not necessary, because the reject() is handled in `Main.tsx`:
https://github.com/neherlab/covid19_scenarios/blob/master/src/components/Main/Main.tsx#L60-L64